### PR TITLE
Normalize description and toString of Infrastructures and Policies for RM portal

### PR DESCRIPTION
**Pull request by AlexFalessi**

### CODE REVIEW

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/BatchJobInfrastructure.java [hunk 1]
```diff
@@ -661,16 +661,15 @@ private void deleteJob(String jobID) throws RMException {
...
+        return getBatchinJobSystemName() + " infrastructure acquires nodes from a resource manager.";
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public String toString() {
-        return getDescription();
+        return getBatchinJobSystemName() + ", maxNodes: [" + this.maxNodes + "], nodeTimeOut: [" + nodeTimeOut + "]";
```
Comment (alijawadfahs) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/BatchJobInfrastructure.java:672]: It's a design choice, but I would rather use string.format() here. more elegant code ;)

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/HostsFileBasedInfrastructureManager.java [hunk 2]
```diff
@@ -592,4 +592,9 @@ private String convertToConfiguredAddress(String compliantAddress) {
 
     }
 
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName() + ", nodeTimeOut: [" + nodeTimeOut + "], maxDeploymentFailure: [" +
```
Comment (alijawadfahs) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/HostsFileBasedInfrastructureManager.java:597]: Here as well String.format() can be useful.

**Review by alijawadfahs — APPROVED** [review]
Minor comments, LGTM

File: rm/rm-policy/rm-policy-scheduler/src/main/java/org/ow2/proactive/scheduler/resourcemanager/nodesource/policy/ReleaseResourcesWhenSchedulerIdle.java [hunk 3]
```diff
@@ -110,7 +110,7 @@ public String getDescription() {
 
     @Override
     public String toString() {
-        return super.toString() + " [idle time: " + idleTime + " ms]";
+        return super.toString() + ", idleTime= " + idleTime + "ms]";
```
Comment (fviale) [inline rm/rm-policy/rm-policy-scheduler/src/main/java/org/ow2/proactive/scheduler/resourcemanager/nodesource/policy/ReleaseResourcesWhenSchedulerIdle.java:113]: idleTime: [...ms] (remove also the ] at the end)

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/BatchJobInfrastructure.java (hunk 1 above)
Comment (fviale) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/BatchJobInfrastructure.java:672]: yes String.format can be used for more elegant code

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/LocalInfrastructure.java [hunk 4]
```diff
@@ -416,7 +416,7 @@ public void shutDown() {
 
     @Override
     public String toString() {
-        return getDescription();
+        return this.getClass().getSimpleName() + ", nodeTimeout: [" + nodeTimeout + "], maxNodes: [" + maxNodes + "]";
```
Comment (fviale) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/LocalInfrastructure.java:419]: we should not put a comma after the infrastructure or policy name, simply a space

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/NodeSourcePolicy.java [hunk 5]
```diff
@@ -267,8 +267,8 @@ public AccessType getUserAccessType() {
      */
     @Override
     public String toString() {
-        return NamesConvertor.beautifyName(this.getClass().getSimpleName()) + " user access type [" + userAccessType +
-               "]" + ", provider access type [" + providerAccessType + "]";
+        return this.getClass().getSimpleName() + ", userAccessType: [" + userAccessType + "]" +
```
Comment (fviale) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/NodeSourcePolicy.java:270]: no comma after the class name

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/LocalInfrastructure.java (hunk 4 above)
Comment (AlexFalessi) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/LocalInfrastructure.java:419]: OK. Applied it on HostsFileBasedInfrastructureManager as well for consistensy

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RestartDownNodesPolicy.java [hunk 6]
```diff
@@ -156,6 +156,14 @@ public String getDescription() {
         return "Static nodes acquisition. If node becomes down policy tries to restart it.";
     }
 
+    /**
+     * Policy string representation.
+     */
+    @Override
+    public String toString() {
+        return super.toString() + ", checkNodeStateEach: [" + this.checkNodeStateEach + "]";
```
Comment (fviale) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RestartDownNodesPolicy.java:164]: why not use String.format also here ?

**Review by fviale — APPROVED** [review]

**Comment by AlexFalessi** [comment]
Retest this please

**Comment by AlexFalessi** [comment]
Retest this please

**Comment by fviale** [comment]
Retest this please

**Comment by fviale** [comment]
Retest this please

**Pull request merged**